packages/markdown/components/Heading/index.jsx

```diff
@@ -1,26 +1,35 @@
 const React = require('react');
 const PropTypes = require('prop-types');
 
-function Heading({ tag, ...props }) {
+function Heading({ tag, opts, ...props }) {
   if (!props.children) return '';
 
+  const { splitReferenceDocs } = opts;
   const attrs = {
     className: `heading heading-${props.level} header-scroll`,
     align: props.align,
   };
-  return React.createElement(tag, attrs, [
+
+  const children = [
     <div key={`heading-anchor-${props.id}`} className="heading-anchor anchor waypoint" id={props.id} />,
     <div key={`heading-text-${props.id}`} className="heading-text">
       {props.children}
     </div>,
-    // eslint-disable-next-line jsx-a11y/anchor-has-content
-    <a key={`heading-anchor-icon-${props.id}`} className="heading-anchor-icon fa fa-anchor" href={`#${props.id}`} />,
-  ]);
+  ];
+
+  if (splitReferenceDocs === 'undefined' || splitReferenceDocs) {
```

Comment:
I'm not sure this would work; wouldn't it need to be a `typeof` checK? In any case, I think we can simplify this logic [if we set the default for this prop to `true`]()!

```suggestion
  if (showAnchorIcons) {
```